.eslintrc.json

```diff
@@ -38,7 +38,12 @@
     "no-invalid-this": "error",
     "no-multiple-empty-lines": ["error", { "max": 1 }],
     "prefer-promise-reject-errors": "error",
-    "@tinymce/prefer-fun": "off"
+    "@tinymce/prefer-fun": "off",
+    "quote-props": [
```

Comment:
Is this a whole-of-tiny lint decision (in which case it probably should go upstream once you've had time to test it out in this repo) or is it a breaking-compatibility-with-rest-of-tiny's-codebases lint decision? I get not using `prefer-fun` because you've got solid reasons for not using katamari et al in this repo, but shouldn't we be aiming for consistency as much as we can both with our other codebases and also the popular standards (read: default eslint settings) within the JS ecosystem?

Reply:
I've raised the suggestion of quote-props with Lee for consideration into our base. He hasn't had a chance to look at it yet. There's a few other rules in here I turned on just to crank up the linting and could be considered in our base rules.

Agree we should aim for consistency within our codebases, but not with popular standards. We tend to be a lot stricter than default eslint settings. 

We need to review our linting settings across all our codebases. Could be worth writing a script to see what's turned on where, what versions of our standard rules are in place.

Reply:
Oof, I forgot about looking into that with 5.7 shenanigans sorry :disappointed:. I'll schedule a time with the editor team and see what they think. My personal vote here is no, I don't like seeing a mix like this and it's why we have all or none atm.

Reply:
I've reverted to TinyMCE settings.